modules/connectivity/ConnectionQuality.js

```diff
@@ -385,42 +332,22 @@ export default class ConnectionQuality {
         } else {
             // Calculate a value based on the sending bitrate.
 
-            // Figure out if simulcast is in use
+            // Figure out if simulcast is in use.
             const activeTPC = this._conference.getActivePeerConnection();
-            const isSimulcastOn
-                = Boolean(activeTPC && activeTPC.isSimulcastOn());
-
-            const newVideoBitrateCap
```

Comment:
BandwidthLimiter was removed here and is no longer applicable for target bitrate calculation - https://github.com/jitsi/lib-jitsi-meet/commit/a1b6f1b8b7aa38b5ce7d21bd5bec5eb8d5760467#diff-dae2c7f98ac6a35c23f8c9684b8b0ce1230f101b061e03b98f7b1303d415224e